packages/test-helpers/ganache-cli.sh

```diff
@@ -7,52 +7,52 @@ set -o errexit
 trap cleanup EXIT
 
 cleanup() {
-  # Kill the RPC instance that we started (if we started one and if it's still running).
+  # Kill the ganache instance that we started (if we started one and if it's still running).
   if [ -n "$rpc_pid" ] && ps -p $rpc_pid > /dev/null; then
     kill -9 $rpc_pid
   fi
 }
 
-setup_coverage_variables() {
-  PORT=${PORT-8555}
-  BALANCE=${BALANCE-100000}
-  GAS_LIMIT=${GAS_LIMIT-0xfffffffffff}
-  NETWORK_ID=${NETWORK_ID-16}
-  ACCOUNTS=${ACCOUNTS-200}
-}
+ganache_port=8545
 
-setup_testing_variables() {
-  PORT=${PORT-8545}
-  BALANCE=${BALANCE-100000}
-  GAS_LIMIT=${GAS_LIMIT-8000000}
-  NETWORK_ID=${NETWORK_ID-15}
-  ACCOUNTS=${ACCOUNTS-200}
+rpc_running() {
+  nc -z localhost "$ganache_port"
 }
 
 rpc_running() {
   nc -z localhost "$PORT"
 }
 
 start_ganache() {
-  echo "Starting ganache-cli..."
-  node_modules/.bin/ganache-cli -i ${NETWORK_ID} -l ${GAS_LIMIT} -a ${ACCOUNTS} -e ${BALANCE} -p ${PORT} > /dev/null &
-  rpc_pid=$!
-  echo "Waiting for ganache to launch on port "$PORT"..."
+  if [ "$SOLIDITY_COVERAGE" = true ]; then
+    export RUNNING_COVERAGE=true
```

Comment:
Interesting, do we not need to set up the coverage-testrpc if we're using builder?